src/eth/miner/block_miner.rs

```diff
@@ -63,15 +65,15 @@ impl BlockMiner {
                 let mined_log = LogMined {
                     log: mined_log,
                     transaction_hash: input.hash.clone(),
-                    transaction_index,
-                    log_index,
+                    transaction_index: transaction_index.clone(),
+                    log_index: log_index.clone(),
                     block_number: block.header.number,
                     block_hash: block.header.hash.clone(),
                 };
                 mined_logs.push(mined_log);
 
                 // increment log index
-                log_index += 1;
+                log_index = log_index + Index::new(1);
```

Comment:
Derive `[Add](https://docs.rs/derive_more/0.99.17/derive_more/#operators)` for `Index`.